Dockerfile

```diff
@@ -11,11 +11,20 @@ RUN apt-get update \
     && apt-get -qy upgrade --fix-missing --no-install-recommends \
     && apt-get -qy install --fix-missing --no-install-recommends \
         curl \
+        git \
```

Comment:
@egabancho, @tiborsimko indeed - it's a discussion moved into git from real life, when I suggested that we can use `python` instead of `python-slim` image and not worry about missing dependencies in the future, if the only benefit of `slim` is saving the disk space. On the second thought, I agree that explicit list of dependencies is a good argument on why we should keep using `python-slim` :)